### PR TITLE
Words Solution

### PR DESCRIPTION
**Pull request by whilestevego**
Submit solution for the words homework. Please the description at certified.in. 🤓 🔥 🚒

### CODE REVIEW

File: math_utils/solution.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
-}
+};
 
-function sub(a, b) {
+const sub = function(a, b) {
   return a - b;
-}
+};
+
+const div = function(a, b) {
```
Comment (whilestevego) [inline math_utils/solution.js]: Good work! Nice div function 😄

File: math_utils/solution.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
-function add(a, b) {
+const add = function(a, b) {
```
Comment (whilestevego) [inline math_utils/solution.js]: Needs more +

Comment (whilestevego) [reply math_utils/solution.js]: Yo

**Comment by whilestevego** [comment]
General comments

File: math_utils/solution.js (hunk 2 above)
Comment (A11good) [reply math_utils/solution.js]: good job Steve!!

**Pull request merged**